python/lsst/summit/utils/astrometry/fastStarTrackerAnalysis.py

```diff
@@ -50,6 +54,29 @@
     "plotSourcesOnImage",
 )
 
+KNOWN_CAMERAS = ("regular", "wide", "fast")
+
+
+@dataclass(frozen=True)
+class StarTrackerCamera:
+    """A frozen dataclass for StarTracker camera configs"""
+
+    cameraType: str
+    suffix: str
+    suffixWithSpace: str
+    doAstrometry: bool
+    cameraNumber: int
+    snr: float
+    minPix: int
+    brightSourceFraction: float
+    scaleError: float
+    doSmoothPlot: bool
+
+
+narrowCam = StarTrackerCamera("regular", "", "", True, 102, 5, 25, 0.95, 5, True)
+wideCam = StarTrackerCamera("wide", "_wide", " wide", True, 101, 5, 25, 0.8, 5, True)
+fastCam = StarTrackerCamera("fast", "_fast", " fast", True, 103, 2.5, 10, 0.95, 60, False)
```

Comment:
Yeah, totally agree, and I usually do use that. I'm also not using `slots` here either, which is a _really_ bad combination! This was because it was all growing very organically and in quite a panic a year ago when we first went on sky. Will tidy that up now 🙏